test/react/rails/railtie_test.rb

```diff
@@ -0,0 +1,14 @@
+require "test_helper"
+
+class RailtieTest < ActionDispatch::IntegrationTest
+  test "reloaders are configured after initializers are loaded" do
+    begin
+      @test_file = File.expand_path("../../dummy/app/pants", File.dirname(__FILE__))
```

Comment:
Good point, I'd forgotten about the file extension. Although some of the tests are passing and they passed locally which is odd as you would expect it not to pass at all. I'll check it out